lib/consts.tsx

```diff
@@ -53,6 +53,21 @@ export const CHANNELS = [
     icon: 'https://i.imgur.com/7ng6bHS.png',
     parentUrl: 'https://en.wikipedia.org/wiki/Johann_Sebastian_Bach',
   },
+  {
+    label: '/electronic',
+    value: 'electronic',
+    icon: 'https://imagedelivery.net/BXluQx4ige9GuW0Ia56BHw/802dff8d-78d2-47cb-9a78-c4fc3accac00/rectcrop3',
+  },
```

Comment:
I think you're missing `parent_hash` for `/electronic`
- I recommend you `remove` electronic from this pull request so you can take more time to properly test `/electronic` while we ship this PR